lesson_07/indx.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" type="text/css" />
+  <title>TerraceTechフロントエンドエンジニア養成所</title>
+</head>
+<body>
+  <div id="js-loading"></div>
```

Comment:
IMO
From the next assignment, it's better to make this with JavaScript. An error occurs when reusing.

Reply:
Yes, sir! I'll do my best.